core-primitives/stf-state-handler/src/file_io.rs

```diff
@@ -19,23 +19,111 @@
 use crate::sgx_reexport_prelude::*;
 
 #[cfg(any(test, feature = "std"))]
-use rust_base58::base58::ToBase58;
+use rust_base58::base58::{FromBase58, ToBase58};
 
 #[cfg(feature = "sgx")]
-use base58::ToBase58;
-
-#[cfg(any(test, feature = "sgx"))]
-use itp_settings::files::ENCRYPTED_STATE_FILE;
+use base58::{FromBase58, ToBase58};
 
 #[cfg(any(test, feature = "sgx"))]
 use std::string::String;
 
-use crate::{error::Result, state_snapshot_primitives::StateId};
-use codec::Encode;
+use crate::{
+	error::{Error, Result},
+	state_snapshot_primitives::StateId,
+};
+use codec::{Decode, Encode};
 use itp_settings::files::SHARDS_PATH;
```

Comment:
This will be removed at the very end of #1292. It is currently also needed in the service to clean-up files, but won't be anymore after that because we will be able to purge the `data-dir` altogether, so the service does no longer need to know the `SHARDS_PATH`.

The above will make the migration of `SHARDS_PATH` to this crate much simpler, so I decided to wait.